src/elements/common/base.scss

```diff
@@ -70,6 +54,32 @@
     }
 }
 
+.be {
+    @include be-content;
```

Comment:
It looks like most of the rules in `be-content` were declared after the `@import` statements below. I'd suggest splitting `@include reset` out:

```
.be {
    @include reset;
    
    // links have tag styles
    @import '../../styles/common/links';
    // Forms have tag styles
    @import '../../styles/common/forms';

    @include be-content;
    // etc.
}
```